src/codegen/codegen_acc_visitor.cpp

```diff
@@ -53,9 +55,11 @@ void CodegenAccVisitor::print_channel_iteration_block_parallel_hint(BlockType ty
 }
 
 
-void CodegenAccVisitor::print_atomic_reduction_pragma() {
-    printer->add_line("nrn_pragma_acc(atomic update)");
-    printer->add_line("nrn_pragma_omp(atomic update)");
+void CodegenAccVisitor::print_atomic_reduction_pragma(bool skip) {
+    if (!skip) {
```

Comment:
I am struggling to understand the point of the `skip` parameter. When is it better to call `print_atomic_reduction_pragma(true)` instead of just deleting the line?

Reply:
you are right! now it is simplified! 

(your comment forced me to look into existing code and realised that #858 has removed OpeMP backend for which empty `print_atomic_reduction_pragma()` calls were added. I got rid of those and now it should be simpler)